src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/DungeonManager.java

```diff
@@ -361,7 +362,15 @@ private static void loadJson(BufferedReader reader, Map<String, JsonElement> map
     }
 
     private static RequiredArgumentBuilder<FabricClientCommandSource, Integer> markSecretsCommand(boolean found) {
-        return argument("secretIndex", IntegerArgumentType.integer()).executes(context -> {
+        return argument("secretIndex", IntegerArgumentType.integer()).suggests((provider, builder) -> {
+            if (isCurrentRoomMatched()) {
+                int secretCount = currentRoom.getSecretCount();
+                for (int i = 1; i < secretCount; i++) {
```

Comment:
Could be a nice stream but whatever.

Reply:
if this is what you meant here you go

Reply:
Wait a minute, this should be inclusive. So `IntStream.rangeClosed(1, room.getSecretCount())` using `rangeClosed` and without the `- 1`.